ovh/ovhcli/order/domains.go

```diff
@@ -67,3 +72,62 @@ var CmdDomain = &cobra.Command{
 
 	},
 }
+
+// CmdDomainTrade in order domain trade
+var CmdDomainTrade = &cobra.Command{
+	Use:   "domainTrade <domain>",
+	Short: "Order domain trade",
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) != 1 {
+			common.WrongUsage(cmd)
+		}
+		domain := args[0]
+
+		if withOwner == "" {
+			err := fmt.Errorf("Missing withOwner option")
```

Comment:
common.Exit("Missing withOwner option") or common.WrongUsage by adding new arg for err description.